src/components/atoms/ImagePicker/Component.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef } from 'react';
+import { Pressable } from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+
+import type { GestureResponderEvent, View } from 'react-native';
+import type {
+  ImageLibraryOptions,
+  ImagePickerResponse,
+} from 'react-native-image-picker';
+
+type ImagePickerLaunchType = 'camera' | 'image-library';
+
+export type OpenPicker = (event: GestureResponderEvent) => void;
+
+export interface ImagePickerProps {
+  launchType: ImagePickerLaunchType;
+  preview: (ImagePickerResponse: ImagePickerResponse) => void;
+  options?: ImageLibraryOptions;
+  children:
+    | React.ReactNode
+    | ((childrenParam: { openPicker: OpenPicker }) => void);
+}
+
+const ImagePicker = forwardRef<View, ImagePickerProps>(
```

Comment:
can we use another library with Compatible with web? like expo-image (can use it over bare project over unimodule, it implemeted already in generated project)